business_requirement_deliverable/README.rst

```diff
@@ -16,6 +16,9 @@ Two new concepts complement the main business requirements model:
 * Deliverable lines
 * Resource lines
 
+A new field for pricelist estimation has been added to the partner to be used in the Deliverable
+lines and Resources lines.
```

Comment:
Since we are describing the module features, instead of "has been added" I would rather say "is available".